src/workerd/api/actor-state.h

```diff
@@ -460,6 +460,10 @@ class DurableObjectState: public jsg::Object {
   // Get the last auto response timestamp or null
   kj::Maybe<kj::Date> getWebSocketAutoResponseTimestamp(jsg::Ref<WebSocket> ws);
 
+  // Gets an array of tags that this websocket was accepted with.
+  // TODO(now): Should this throw if you provide a non-hibernatable websocket?
```

Comment:
I think the two options are to return an empty array or to throw? If someone can reasonably determine if a web socket is hibernatable or not, I'd say throw. If not return an empty array to make this code easier to work with.